packages/generate-EN10168-pdf-template/test/renderingCommTrans.ts

```diff
@@ -0,0 +1,86 @@
+import { expect } from 'chai';
+
+const certificate = require('./testingCertificate.json');
```

Comment:
Please use the resources in fixtures folder, v0.0.2 is the version you build the pdf-template generator for.